internal/services/service_test.go

```diff
@@ -0,0 +1,147 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package services
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/cartesi/rollups-node/internal/logger"
+	"github.com/stretchr/testify/suite"
+)
+
+type ServicesTestSuite struct {
+	suite.Suite
+	tmpDir      string
+	servicePort int
+}
+
+func (s *ServicesTestSuite) SetupSuite() {
+	logger.Init("warning", false)
+	s.buildFakeService()
+	s.servicePort = 55555
+}
+
+func (s *ServicesTestSuite) TearDownSuite() {
+	err := os.RemoveAll(s.tmpDir)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
I hadn't thought about using assert/require for functions that are not proper tests. After trying it I found that using `panic` provides a better description of what happened since it prints the error message instead of the error object itself. 

I would rather keep this as is, but we can discuss this further if you like.